openedx/features/course_search/static/course_search/js/spec/course_search_spec.js

```diff
@@ -252,18 +252,18 @@ define([
                 expect(this.item.$el).toHaveAttr('role', 'region');
                 expect(this.item.$el).toHaveAttr('aria-label', 'search result');
                 expect(this.item.$el).toContainElement('a[href="' + this.model.get('url') + '"]');
-                expect(this.item.$el.find('.result-type')).toContainHtml(this.model.get('content_type'));
+                expect(this.item.$el.find('.result-type i')).toHaveClass('fa-' + this.item.unitIcon());
                 expect(this.item.$el.find('.result-excerpt')).toContainHtml(this.model.get('excerpt'));
-                expect(this.item.$el.find('.result-location')).toContainHtml('section ▸ subsection ▸ unit');
+                expect(this.item.$el.find('.result-link')).toContainHtml('section ▸ subsection ▸ unit');
             });
 
             it('rendersSequentialItem', function() {
                 expect(this.seqItem.$el).toHaveAttr('role', 'region');
                 expect(this.seqItem.$el).toHaveAttr('aria-label', 'search result');
                 expect(this.seqItem.$el).toContainElement('a[href="' + this.seqModel.get('url') + '"]');
-                expect(this.seqItem.$el.find('.result-type')).toBeEmpty();
+                expect(this.seqItem.$el.find('.result-type i')).toHaveClass('fa-' + this.seqItem.unitIcon());
                 expect(this.seqItem.$el.find('.result-excerpt')).toBeEmpty();
-                expect(this.seqItem.$el.find('.result-location')).toContainHtml('section ▸ subsection');
+                expect(this.seqItem.$el.find('.result-link')).toContainHtml('section ▸ subsection');
```

Comment:
We'll need to update the expected separator here (and elsewhere).

Reply:
Updated test cases.